sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -52,6 +52,7 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
 
             var itemCardViewOneCol = new Sourcemash.Views.ItemCardView({el: "#item-" + item.get('id') + "-onecol", model: item, user: user });
             itemCards.push(itemCardViewOneCol);
+            $(".card-reveal a").attr("target","_blank");
```

Comment:
But we want to apply this only to the contents of the card reveal, which we won’t know what it is since we scrape it...